### DISCUSSION TITLE
Table flickers when refreshing from provider due to busy state

### DISCUSSION THREAD

**Discussion by puigru** [bootstrap-vue/bootstrap-vue#5888] [Q&A] [answered]
I have a table with the `items` prop set to a provider function which pulls data from an API, then have a `setInterval` in `mounted` which calls `this.$refs.table.refresh()` as suggested in #2144. However, every time the table is updated, there is noticeable flicker.

![flicker](https://user-images.githubusercontent.com/7153662/95794572-851abc80-0ce8-11eb-8102-36a9c74aeed2.gif)

This seems to occur due to this styling being applied to the table while retrieving the response from the server:
```css
.table.b-table[aria-busy="true"] {
  opacity: 0.55;
}
```

How do I avoid this?

**Comment by Hiws** [comment, marked as answer]
You can simply change the CSS to be `opacity: 1`, overwriting the styling you mentioned.

If you want to apply it globally you can change the SCSS variable `$b-table-busy-opacity`.
